ha/const.py

```diff
@@ -64,9 +65,11 @@
 SERVICE_COMMAND="service"
 BUNDLE_COMMAND="support_bundle"
 
+PCS_CLUSTER_PACKAGES=["pacemaker", "corosync", "pcs"]
 PCS_CLEANUP="pcs resource cleanup"
 PCS_FAILCOUNT_STATUS="pcs resource failcount show"
-PCS_STATUS="pcs status"
+PCS_STATUS = "pcs status"
+PCS_CLUSTER_DESTROY="pcs cluster destroy"
```

Comment:
destroy has "-all" option. It may not be required for single node VM but will be required in multi-node for cleanup stage.

Reply:
--all command fails currently due to some reason, hence not adding --all option for m0.